hera_cal/abscal.py

```diff
@@ -1074,6 +1139,11 @@ def abscal_run(data_file, model_files, refant=None, calfits_infile=None, verbose
 
     max_dlst : type=float, maximum allowed LST difference in model rephasing, otherwies model is flagged.
 
+    window : str, window to enact on data before FFT for delay solver, options=['blackmanharris', 'hann', None]
+        None is a top-hat window.
+
+    edge_cut : int, number of channels to flag at each band edge in delay solvers
```

Comment:
"flag" is a bit confusing, since we're not actually applying flags to that data. I'd say "exclude".